extensions/mssql/src/sqlToolsServer.ts

```diff
@@ -217,7 +217,8 @@ function getClientOptions(context: AppContext): ClientOptions {
 	return {
 		documentSelector: ['sql'],
 		synchronize: {
-			configurationSection: Constants.extensionConfigSectionName
+			configurationSection: [Constants.extensionConfigSectionName,
```

Comment:
nit: Please format this so the config sections are on separate lines and lined up nicely